numcodecs/tests/test_astype.py

```diff
@@ -63,7 +63,7 @@ def test_backwards_compatibility():
     # integers
     arrs = [
         np.arange(1000, dtype='<i4'),
-        np.random.randint(0, 200, size=1000, dtype='<i4').reshape(100, 10, order='F'),
+        np.random.randint(0, 200, size=1000, dtype='i4').astype('<i4').reshape(100, 10, order='F'),
```

Comment:
```suggestion
        np.random.randint(0, 200, size=1000, dtype='<i4').reshape(100, 10, order='F'),
```

Could simplify.

Reply:
No, see the commit message; `np.random.randint` does not accept non-native byteorder.